_includes/head.html

```diff
@@ -23,7 +23,7 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ BASE_URL }}/assets/css/main.css">
-
+<link rel="stylesheet" href="{{ BASE_URL }}/assets/css/custom.css">
```

Comment:
Let's use Sass! I would create a file called `_nav.scss` in the scss folder, and then import it there. It's a lot more powerful than CSS, and makes things more streamlined. That's why you couldn't find the CSS folder - that's where I've been editing stuff.

Reply:
I think I need a little bit more clarification on this. I understand the part about creating '_nav.scss' in the '_sass' folder.

Are you saying I should create another file called 'custom.css' in the '_sass' folder? 

Then do something like '@import "custom.css" inside '_nav.scss' ?

Reply:
Don't create a `.css` file at all. Jekyll will automatically create it for you. import `_nav` in `main.scss`. :)